### PR TITLE
Update coherence operator to v3.2.2

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

Uptake Coherence operator v.3.2.2. As part of this change, verrazzano-operator version is updated to get the Coherence dashboards from v.3.2.2.

Fixes VZ-3482

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/verrazzano-bom.json [hunk 1]
```diff
@@ -221,7 +221,7 @@
           "images": [
             {
               "image": "verrazzano-operator",
-              "tag": "1.1.0-20210721203425-f4d8b23",
+              "tag": "1.1.0-20210907141728-f037a71",
```
Comment (bobdonat) [inline platform-operator/verrazzano-bom.json:224]: how does this change relate to updating coh operator

**Review by bobdonat — APPROVED** [review]

File: platform-operator/thirdparty/charts/coherence-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -16,7 +16,7 @@ metadata:
     control-plane: coherence
     app.kubernetes.io/name: coherence-operator
     app.kubernetes.io/instance: coherence-operator-manager
-    app.kubernetes.io/version: "3.2.1"
+    app.kubernetes.io/version: "3.2.2"
```
Comment (pfmackin) [inline platform-operator/thirdparty/charts/coherence-operator/templates/deployment.yaml:19]: Did you replace the entire coherence-operator chart?

File: platform-operator/helm_config/overrides/coherence-values.yaml [hunk 3]
```diff
@@ -3,4 +3,4 @@
 
 # NOTE: The image you're looking for isn't here. The coherence-operator image now comes from
 # the bill of materials file (verrazzano-bom.json).
-defaultCoherenceImage: ghcr.io/oracle/coherence-ce:21.06
+defaultCoherenceImage: ghcr.io/oracle/coherence-ce:21.06.1
```
Comment (pfmackin) [inline platform-operator/helm_config/overrides/coherence-values.yaml:6]: This field should be removed.  Change the value in the BOM

File: platform-operator/thirdparty/charts/coherence-operator/templates/deployment.yaml (hunk 2 above)
Comment (bhatpmk) [reply platform-operator/thirdparty/charts/coherence-operator/templates/deployment.yaml:19]: Yes, I have replaced all the files.

File: platform-operator/helm_config/overrides/coherence-values.yaml (hunk 3 above)
Comment (bhatpmk) [reply platform-operator/helm_config/overrides/coherence-values.yaml:6]: I have updated the BOM, but this change is already submitted. I will remove this in a separate PR.

**Pull request merged**